modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -332,6 +333,13 @@ public void start(@Nullable String cfg) {
             // Deploy all registered watches because all components are ready and have registered their listeners.
             metaStorageMgr.deployWatches();
 
+            baselineMgr.listenBaselineChange(ctx -> {
+                // TODO: Multi invoke supposed to be here https://issues.apache.org/jira/browse/IGNITE-16063
+                metaStorageMgr.put(new ByteArray("some key"), new byte[0]);
```

Comment:
It would be nice to change the name of the key even though it is a "temporary" name. Perhaps, "baseline.nodes".

Reply:
I could change it, if we decide to merge it not only to the feature branch, but to the main also

Reply:
> I could change it, if we decide to merge it not only to the feature branch, but to the main also
In any case, the name of a property should be meaningful. IMHO, `some key` is not good enough for the feature branch as well.